zui/src/transforms/zui-to-typescript-next/index.test.ts

```diff
@@ -0,0 +1,594 @@
+import { describe, it, expect } from 'vitest'
+import { toTypescript } from '.'
+import z from '../../z'
+
+describe('functions', () => {
+  it('title mandatory to declare', async () => {
+    const fn = z
+      .function()
+      .args(z.object({ a: z.number(), b: z.number() }))
+      .returns(z.number())
+      .describe('Add two numbers together.\nThis is a multiline description')
+    expect(() => toTypescript(fn, { declaration: true })).toThrowError(/title/i)
```

Comment:
**nitpick:** the `toTypescript` could export the errors it throws so you can assert an error type instead of an error message here (it's more precise)

Reply:
very true, will fix

Reply:
changed as pointed out